src/perf/impeller.md

```diff
@@ -74,6 +74,44 @@ include a small reproducible test case.
 
 [file-issue]: {{site.github}}/flutter/flutter/issues/new/choose
 
+### macOS
```

Comment:
Does Impeller work on iOS mobile as well as macOS desktop? I see both used here.

Reply:
Yes. We are moving to preview Impeller on macOS desktop.

Reply:
Yeah we already have iOS support, macOS preview is being added in 3.13

Reply:
Great!